tests/test_bs.py

```diff
@@ -82,25 +96,67 @@ def add_multi_clients(deployment_id, container_specs, size=2):
                                                         include_uninitialized=True,
                                                         label_selector=("name={0}".format(
                                                             resp.metadata._name.split('-')[1]))).items
-    pods = []
-    for c in client_pods:
-        pod_name = c.metadata.name
-        if pod_name.startswith(resp.metadata.name):
-            pods.append(pod_name)
-    return pods
+    if ret_pods:
+        ret_val = client_pods
```

Comment:
Simply write: return client_pods

Reply:
this way is easier to debug + I always inspire to have one return line in each function